scripts/update-grafana-version.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
```

Comment:
The updates to package.json, docker and plugin.json files were made by this script.